src/App/Fossa/ReleaseGroup/Create.hs

```diff
@@ -0,0 +1,30 @@
+{-# LANGUAGE RecordWildCards #-}
+
+module App.Fossa.ReleaseGroup.Create (
+  createMain,
+) where
+
+import App.Fossa.Config.ReleaseGroup.Create (CreateConfig (..))
+import Control.Algebra (Has)
+import Control.Effect.Diagnostics (Diagnostics)
+import Control.Effect.FossaApiClient (FossaApiClient, createReleaseGroup, getOrganization)
+import Control.Effect.Lift (Lift)
+import Control.Monad (when)
+import Data.String.Conversion (ToText (..))
+import Effect.Logger (Logger, logInfo, logStdout)
+import Fossa.API.Types (CreateReleaseGroupResponse (..), Organization (orgSupportsReleaseGroups))
+
+createMain ::
+  ( Has Diagnostics sig m
+  , Has Logger sig m
+  , Has (Lift IO) sig m
+  , Has FossaApiClient sig m
+  ) =>
+  CreateConfig ->
+  m ()
+createMain CreateConfig{..} = do
+  logInfo "Running FOSSA release-group create"
+  org <- getOrganization
+  when (orgSupportsReleaseGroups org) $ do
```

Comment:
[required] we should throw error (exit code non-zero), and report to user how they can enable this functionality, instead of silently exiting!